ci/arm/gitlab-ci.yml

```diff
@@ -0,0 +1,10 @@
+linux-arm64:
```

Comment:
Do we need multiple jobs for musl vs glibc?

Reply:
the azure script didn't have that, so I didn't think of it.